braindecode/models/util.py

```diff
@@ -54,6 +59,10 @@ def to_dense_prediction_model(model, axis=(2, 3)):
             module.stride = tuple(new_stride)
 
 
+@deprecated(
+    "will be removed in version 1.0. Use EEGModuleMixin.get_output_shape method directly on the "
+    "model object."
+)
 def get_output_shape(model, in_chans, input_window_samples):
```

Comment:
same comment with `model.get_output_shape()`

Reply:
the same as for the previous comment